apps/builder/app/builder/shared/code-editor-base.tsx

```diff
@@ -420,14 +420,14 @@ export const EditorDialog = ({
 export const CodeEditorBase = ({
```

Comment:
Everything inside this component already reusable and composable. There is no sense in this component as abstraction like this.

Reply:
done